projects/packages/changelogger/tests/php/tests/src/AddCommandTest.php

```diff
@@ -399,6 +399,19 @@ public function provideExecute() {
 					'/Entry must be specified in non-interactive mode/',
 				),
 			),
+			'Non-interactive use with missing entry and significance of patch' => array(
+				array(
+					'--significance' => 'patch',
+					'--type'         => 'fixed',
+				),
+				array( 'interactive' => false ),
+				array(),
+				1,
+				null,
+				array(
+					'/Entry must be specified in non-interactive mode./\nIf you want to have an empty entry for this change, pass the empty string as the entry (like --entry=) and also please provide a comment (using --comment)./',
```

Comment:
Either like this
```suggestion
					'/Entry must be specified in non-interactive mode\.\nIf you want to have an empty entry for this change, pass the empty string as the entry \(like --entry=\) and also please provide a comment \(using --comment\)\./',
```
or like this
```suggestion
					'/Entry must be specified in non-interactive mode\./',
					'/If you want to have an empty entry for this change, pass the empty string as the entry \(like --entry=\) and also please provide a comment \(using --comment\)\./',
```